.tx/config

```diff
@@ -1,8 +1,8 @@
 [main]
 host = https://www.transifex.com
 
-[GLPI.GLPIINventoryPlugin.glpipot10]
+[glpi-plugin-glpiinventory.glpiinventory-pot]
 file_filter = locales/<lang>.po
 source_file = locales/glpiinventory.pot
-source_lang = en_GB
-minimum_perc: 100
```

Comment:
Maybe should we keep a minimum percent valule, let's say to 10% (that's the value I've used for years on glpi tx pulls).